docs-ref-conceptual/azure-cli-support-lifecycle.md

```diff
@@ -0,0 +1,194 @@
+  "azure-cli-support": "LTS-20240521",  
+  "azure-cli-telemetry": "1.0.8",  
+  "extensions": {  
+  }  
+}  
+```
+
+The following image illustrates the release cadence and support lifecycle of Azure PowerShell for LTS and STS releases.
+
+![Support Lifecycle](./media/support-lifecycle/support-lifecycle.png)
+
+The following table provides examples of how the release cadence correlates with the support
+lifecycle:
+
+> [!NOTE]
+> The dates in the following table are for example purposes only. The actual release dates vary.
+
+| Version number | Release schedule | Breaking changes | Release type | Support type | Supported until version |
+| -------------- | ---------------- | ---------------- | ------------ | ------------ | ----------------------- |
+| 3.0.0          | May 2024         | ![Yes][1]        | Major        | LTS          | 3.1.0                  |
+| 3.1.0          | June 2024        |                  | Minor        | LTS          | 3.2.0                  |
+| 3.2.0          | July 2024        |                  | Minor        | LTS          | 3.3.0                  |
+| 3.3.0          | August 2024      |                  | Minor        | LTS          | 3.4.0                  |
+| 3.4.0          | September 2024   |                  | Minor        | LTS          | 3.5.0                  |
+| 3.5.0          | October 2024     |                  | Minor        | LTS          | _**6.0.0**_            |
+| 4.0.0          | November 2024    | ![Yes][1]        | Major        | STS          | 4.1.0                  |
+| 4.1.0          | December 2024    |                  | Minor        | STS          | 4.2.0                  |
+| 4.2.0          | January 2025     |                  | Minor        | STS          | 4.3.0                  |
+| 4.3.0          | February 2025    |                  | Minor        | STS          | 4.4.0                  |
+| 4.4.0          | March 2025       |                  | Minor        | STS          | 4.5.0                  |
+| 4.5.0          | April 2025       |                  | Minor        | STS          | _**7.0.0**_            |
+| 5.0.0          | May 2025         | ![Yes][1]        | Major        | LTS          | 5.1.0                  |
+
+- Examples:
+  - To qualify for support, you must update to 3.1.0 when released if you have Azure CLI 3.0.0 installed.
+  - If Azure CLI 3.5.0 is the final release of Azure CLI version 3, it's supported until the release of Azure CLI
+    6.0.0.
+  - When Azure CLI 5.0.0 is released, Azure CLI version 4 is no longer supported.
+
+## Breaking changes
+
+Major version releases of Azure CLI introduce breaking changes and occur up to two times per
+year. Breaking changes may occur at any point for preview extensions. Non-GA extensions aren't
+required to adhere to breaking change policies.
+
+For detailed information about breaking changes in Azure CLI, see [Release notes](./release-notes-azure-cli.md).
+
+## Supported environments
```

Comment:
@mikefrobbins , This sections feels redundant to #operating-systems below

Reply:
Consider combining the two.